sdk/mobile-center/src/test/java/com/microsoft/azure/mobile/MobileCenterTest.java

```diff
@@ -212,6 +222,7 @@ public void configureAndStartTwiceTest() {
         verify(service).getLogFactories();
         verify(service).onChannelReady(any(Context.class), notNull(Channel.class));
         verify(application).registerActivityLifecycleCallbacks(service);
+        verify(mChannel).enqueue(any(StartServiceLog.class), eq(MobileCenter.CORE_GROUP));
```

Comment:
Given the feature, this is not good enough verification in general in that class, you should mock whenNew for StartService logs and check what arguments are passed to setServices, especially to check for already started service tests.